File: Strings/Java/Count_type_of_Characters.java
class Sol
{
    int[] count (String s)
    {
        // your code here   
        int arr[] = new int [4];
        for (int i=0; i<s.length(); i++) {
            char c = s.charAt(i);
            if (c>='A' && c<='Z') {
                arr[0]++;
            }
            else if (c>='a' && c<='z') {
                arr[1]++;
            }
            else if (Character.getNumericValue(c)>=0 && Character.getNumericValue(c)<=9) {
                arr[2]++;
            }
            else {
                arr[3]++;
            }
       }
       return arr;
    }
}

File: Solutions/Python/Count_type_of_Characters.py
class Solution:
    def count (self,s):
        # your code here
        count = [0, 0, 0, 0]
        for i in s:
            if i.isupper():
                count[0] += 1
            elif i.islower():
                count[1] += 1
            elif i.isdigit():
                count[2] += 1
            else:
                count[3] += 1
        return count
